web/src/components/molecules/Common/Form/AssetItem/hooks.ts

```diff
@@ -30,10 +32,10 @@ export default (
 
   const handleUpload = useCallback(async () => {
     setUploading(true);
-    await createAssets(fileList);
+    const assets = await createAssets(fileList);
+    if (assets?.length > 0) onLink(assets[0]);
```

Comment:
Shouldn't we limit to only a single asset can be created in this form? Or, have you talked to @lavalse about this and you both feel this approach to add multiple, but select the first one, makes sense?

Reply:
I am using the same fileUpload modal in asset list page and in schema and item. both use the same method `createAssets` which takes an array of files as an argument and returns an array of assets as a result.
However, in schema and item I already added `multiple: false` and `maxCount: 1` in the modal props, so it doesn't allows multiple file upload at.
please check [this](https://github.com/reearth/reearth-cms/blob/f1cc6f125ab70fd68a2e7b0f949461f910d928be/web/src/components/molecules/Common/Form/AssetItem/index.tsx#L59) for reference.

Reply:
Oh ok! Sounds good